sdk/keyvault/azure-keyvault-secrets/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 4.2.0b3 (Unreleased)
+- All long running operation methods now accept a `continuation_token` keyword argument
```

Comment:
I'd be more specific here, if only to avoid implying async methods are affected.

Reply:
Makes sense, changing now, thanks!